src/biome/text/pipeline.py

```diff
@@ -711,23 +723,45 @@ def create_vocabulary(self, config: VocabularyConfiguration) -> None:
         config
             Specifies the sources of the vocabulary and how to extract it
         """
+        # TODO: Do we have to create an empty vocab here, can't we just use the current pipeline vocab?
+        #       If we use the pipeline vocab, the call to `self._add_transformers_vocab` is not necessary!
```

Comment:
@frascuchon Can't we just use the `self._model.vocab` here instead of creating a new empty vocab (via `self._create_empty_vocab`)?

Reply:
Yes, we can pass the model vocab. 

```python
vocab = self._extend_vocabulary(self._model.vocab, config)
```